solver/result/attestation.go

```diff
@@ -11,7 +11,8 @@ type Attestation struct {
 	Ref  string
 	Path string
 
-	InToto InTotoAttestation
+	InToto            InTotoAttestation
+	PredicateCallback func() ([]byte, error)
```

Comment:
I'd prefer if we call this `Callback`, and keep it generic in naming along with the `Ref` and `Path`.

Reply:
I think `Callback` is too generic. Maybe `ContentCallback` or `ContentFunc`?

Reply:
`ContentFunc` sounds good to me :+1: